src/main/java/io/anserini/search/topicreader/NtcirTopicReader.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Anserini: A Lucene toolkit for replicable information retrieval research
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.search.topicreader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * Topic reader for the XML format used in the NTCIR We Want Web (WWW) Tracks.
+ * Please cite the following papers if the NTCIR WWW test collections are used
+ * in a publication.
+ *
+ * http://research.nii.ac.jp/ntcir/workshop/OnlineProceedings13/pdf/ntcir/01-NTCIR13-OV-WWW-LuoC.pdf
+ * http://research.nii.ac.jp/ntcir/workshop/OnlineProceedings14/pdf/ntcir/01-NTCIR14-OV-WWW-MaoJ.pdf
+ */
+public class NtcirTopicReader extends TopicReader<Integer> {
+    public NtcirTopicReader(Path topicFile) {
```

Comment:
Two space indent please.

Reply:
Thank you for your review. I have fixed the indent..